src/mixins/allowable-actions-mixin.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, Vue } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { getFeatureFlag } from '@/utils'
+import { AllowableActions, CorpTypeCd } from '@/enums'
+
+@Component({})
+export default class AllowableActionsMixin extends Vue {
+  @Getter getEntityType!: CorpTypeCd
+  @Getter hasBlocker!: boolean
+  @Getter isBComp!: boolean
+  @Getter isBcCompany!: boolean
+  @Getter isUlc!: boolean
+  @Getter isHistorical!: boolean
+  @Getter isRoleStaff!: boolean
+
+  /**
+   * Returns True if the specified action is allowed, else False.
+   * @param action the action to check
+   */
+  isAllowed (action: AllowableActions): boolean {
+    const businessId = sessionStorage.getItem('BUSINESS_ID')
```

Comment:
This CANNOT be a getter as it would not be reactive. I found this out the hard way.